.buildkite/pipeline.yml

```diff
@@ -0,0 +1,31 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+---
+
+# Variables used in this pipeline are defined in `shared-pipeline-vars`, which is `source`'d before calling `buidkite-agent pipeline upload`
+
+agents:
+  queue: "mac"
+env:
+  IMAGE_ID: $IMAGE_ID
+
+steps:
+  - label: ":swift: SwiftFormat Linting"
+    key: swiftformat-lint
+    plugins:
+      - docker#v5.12.0:
+          image: "ghcr.io/nicklockwood/swiftformat:$SWIFTFORMAT_VERSION"
+          command: [ "GravatarApp", "GravatarAppTests", "Packages", "--lint" ]
+          workdir: "${BUILDKITE_BUILD_CHECKOUT_PATH}"
```

Comment:
💭  Note that on one end I like that we run this in `docker` and leverage the existence of the dedicated swiftformat docker image 👍 ❤️  but otoh I realized this might need to be kept in sync with how it's been introduced via `make lint` in https://github.com/Automattic/Gravatar-iOS/pull/2, so maybe someone from the Gravatar dev team can chime in there to share their thoughts on how they plan to use Swiftlint and how we could keep the configs in sync?